Ginger/GingerCoreNET/Run/RunListenerLib/SealightsExecutionLogger/SealightsReportExecutionLogger.cs

```diff
@@ -233,6 +233,17 @@ public override async void ActionEnd(uint eventTime, Act action, bool offlineMod
         }
 
 
-        #endregion Action       
+        #endregion Action  
+
+        public override async void ActivitySkippedEnd(uint eventTime, Activity activity, bool offlineMode = false)
+        {
+            await SealightsReportApiHandler.SendingTestEventsToSealightsAsync(activity.ActivityName, activity.StartTimeStamp, activity.EndTimeStamp, "Skipped");
+        }
+
+        public override async void ActivityGroupSkippedEnd(uint eventTime, ActivitiesGroup activityGroup, bool offlineMode = false)
+        {
+            await SealightsReportApiHandler.SendingTestEventsToSealightsAsync(activityGroup.Name, activityGroup.StartTimeStamp, activityGroup.EndTimeStamp, "Skipped");
+        }
```

Comment:
what about handling BF Skipped?

Reply:
BF is handling/capturing the "skipped" event in the existing/original code already